### ISSUE TITLE
Need an aria-datetime attribute for parity with HTML datetime attr

### ISSUE THREAD

**Issue by melanierichards** [w3c/aria#898]
Related to #878 (`time` role)

When reviewing PR #895 for the new `time` role, the group determined that an `aria-datetime` attribute would be a good addition when we embark on attribute parity. `aria-datetime` would be the equivalent of the HTML `datetime` attribute: https://www.w3.org/TR/html52/textlevel-semantics.html#element-attrdef-time-datetime

Authors use `datetime` to provide the date/time in a machine-readable format; they should use `aria-datetime` to do the same. `aria-datetime` should be a supported (not required) property of the `time` role.

**Comment by smhigley** [comment]
For reference, here are the HTML AAM mappings for `datetime` on `<time>`: https://w3c.github.io/html-aam/#att-datetime-time

**Comment by jnurthen** [comment]
I'm confused why this might be useful. Can anyone explain to me why AT needs to know the contents of something whose intended purpose is to be machine readable?

**Comment by jnurthen** [comment]
closing for now... please reopen if anyone thinks it might be useful
